web/src/css/media_queries.scss

```diff
@@ -0,0 +1,52 @@
+/*
+Add media queries/breakpoints related rules here.
+*/
+@media screen and (max-width: 640px) {
+    .d-md-block {
+      display: block;
+    }
+    .d-md-none {
+      display: none;
+    }
+  
+    .subtitle {
+      flex-direction: column;
+      text-align: center;
+  
+      h5 {
+        line-height: 1.5rem;
+      }
+    }
+  
+    .mdc-data-table {
+      .mdc-data-table__header-cell,
+      .mdc-data-table__cell {
+        max-width: 100px;
```

Comment:
If I remove this rule, the "Global HTTP load balancer" doesn't wrap and looks better.

Reply:
Before:

![image](https://user-images.githubusercontent.com/1175430/151604620-2ff6c31e-853f-432b-8f38-e207e5979d30.png)


After:

![image](https://user-images.githubusercontent.com/1175430/151604580-88802e3a-f3f9-42bb-89e1-416580768e9b.png)